src/database/migrations/20240110072148-seed-data-to-notification-template.js

```diff
@@ -0,0 +1,80 @@
+'use strict'
+/** @type {import('sequelize-cli').Migration} */
+let addedRecordIds = [11, 12, 13, 14] // Keep track of added record IDs
+
+module.exports = {
+	up: async (queryInterface, Sequelize) => {
+		const defaultOrgId = queryInterface.sequelize.options.defaultOrgId
+		if (!defaultOrgId) {
+			throw new Error('Default org ID is undefined. Please make sure it is set in sequelize options.')
+		}
+
+		return queryInterface.bulkInsert('notification_templates', [
+			{
+				id: 11,
```

Comment:
@VISHNUDAS-tunerlabs if id is autonumeric, then if the data present already for that, it will be a problem so, don't provide this

Reply:
@rakeshSgr I have tried to insert data without providing id initially. But I was getting error. Even if id is auto incremented . I was getting : Validation error
ERROR DETAIL: Key (id, organization_id)=(1, 1) already exists. 

that why keep it like this

Reply:
@VISHNUDAS-tunerlabs Please fix this.  It will be a problem, if we have already existing records  for 11, 12, 13, 14, 15, 16, 17

Reply:
@rakeshSgr Ok. I am checking if we can do this dynamically